extension/src/background/messageListener/freighterApiMessageListener.ts

```diff
@@ -171,21 +155,7 @@ export const freighterApiMessageListener = (
       const domain = getUrlHostname(tabUrl);
       const punycodedDomain = getPunycodedDomain(domain);
 
-      const networkDetails = await getNetworkDetailsFromNetworkpassphrase(
-        networkPassphrase,
-      );
-
-      const allowListSegment = await getAllowListSegment({
-        publicKey,
-        networkDetails,
-      });
-      const isDomainListedAllowed = isSenderAllowed({
-        sender,
-        allowListSegment,
-      });
-
       const tokenInfo: TokenToAdd = {
-        isDomainListedAllowed,
```

Comment:
as mentioned in the commit description, we are moving this check to the UI where it can be calculated dynamically